hydra/test_utils/configs/conf/config.yaml

```diff
@@ -0,0 +1,2 @@
+# @package _global_
+normal_yaml_config: true
```

Comment:
Is this accidental? remove the directory?

Reply:
no, otherwise 1 integration tests to override the config dir will break

Reply:
since now it looks under conf/ directory

Reply:
Previously I think I tried to add base_module in TARGETS, it didn't work. Maybe I was not doing it properly then

Reply:
See `fbcode/github/facebookresearch/hydra/examples/tutorials/basic/your_first_hydra_app/2_config_file/TARGETS` for an example of an app with a config file.

```
$ cat  examples/tutorials/basic/your_first_hydra_app/2_config_file/TARGETS
python_binary(
    name = "my_app",
    srcs = glob(["**/*.py"]),
    # this particular example requires a module because the config module is empty
    base_module = "module",
    main_module = "module.my_app",
    # Add config files to resources for them to be available to Hydra.
    resources = glob(["**/*.yaml"]),
    deps = [
        "//github/facebookresearch/hydra:hydra",
    ],
)
```